src/server/jobs/prepare-bounties.ts

```diff
@@ -7,28 +7,158 @@ import { createBuzzTransaction } from '~/server/services/buzz.service';
 import { TransactionType } from '~/server/schema/buzz.schema';
 import { Tracker } from '../clickhouse/client';
 import { handleTrackError } from '../utils/errorHandling';
+import {
+  bountyAutomaticallyAwardedEmail,
+  bountyExpiredEmail,
+  bountyExpiredReminderEmail,
+  bountyRefundedEmail,
+} from '~/server/email/templates';
 
-const log = createLogger('bounties', 'blue');
+const log = createLogger('prepare-bounties', 'blue');
 
-const prepareBounties = createJob('prepare-bounties', '0 1 * * *', async () => {
+const prepareBounties = createJob('prepare-bounties', '0 23 * * *', async () => {
   const [lastRun, setLastRun] = await getJobDate('prepare-bounties');
+  const justExpiredBounties = await dbWrite.bounty.findMany({
+    where: {
+      complete: false,
+      // Expires today
+      expiresAt: dayjs().toDate(),
```

Comment:
Wouldn't this match exactly time and date? Maybe condition this between today start of day and end of day?

Reply:
The table has been updated to only support date as we don't care for time :D